src/ontology/mondo-edit.obo

```diff
@@ -245209,17 +245210,21 @@ synonym: "progressive epilepsy - intellectual disability, Finnish type" RELATED
 synonym: "progressive epilepsy with intellectual disability, northern epilepsy" EXACT [DOID:0110724]
 synonym: "progressive epilepsy with mental retardation, northern epilepsy" EXACT DEPRECATED [DOID:0110724]
 synonym: "progressive epilepsy-intellectual disability syndrome, Finnish type" EXACT [DOID:0110724, GARD:0004010]
+synonym: "progressive myoclonic epilepsy with neuroserpin inclusion bodies" EXACT [MONDO:0035160]
 xref: DOID:0110724 {source="MONDO:equivalentTo"}
 xref: ICD10CM:E75.4 {source="DOID:0110724", source="Orphanet:1947/attributed", source="Orphanet:1947/ntbt", source="Orphanet:1947"}
+xref: ICD10CM:G40.3 {source="MONDO:mondoIsNarrowerThanSource", source="Orphanet:530298"}
 xref: OMIM:610003 {source="Orphanet:1947/e", source="DOID:0110724", source="MONDO:equivalentTo", source="Orphanet:1947"}
 xref: Orphanet:1947 {source="OMIM:610003", source="DOID:0110724", source="MONDO:equivalentTo", source="GARD:0004010"}
+xref: Orphanet:530298
```

Comment:
This is missing `{source="MONDO:equivalentTo"}` from `MONDO:0035160`. Was this dropped using "owltools"... is this an existing bug?

Reply:
I purposely removed it so we didn't have a proxy merge. Ada advised me to merge these terms but Orphanet has not merged them.